spec/models/user_spec.rb

```diff
@@ -335,4 +335,17 @@
     end
   end
 
+  describe 'before_save :check_phone_verification' do
+    let(:user) { create(:user) }
+    before { user.update!(verified: true) }
```

Comment:
Can we just?

`let(:user) { create(:user, verified: true) }`

Reply:
This does seem cleaner, however I tried it and for some reason it doesn't work... verified is still false in the created object. Trying to figure out why

Reply:
Oh, probably because my after_save callback is setting it to false when the phone number is first set.